src/services/http.service.ts

```diff
@@ -82,9 +82,9 @@ class HttpService {
     const reqUrl: string = this.urlIsAbsolute(params.origUrl) ? params.origUrl : this.baseUrl + params.origUrl;
     const finalUrl: string = this.buildUrl(reqUrl, params.queryParams, params.rawData);
     const requestParams = {
-      method: params.method || 'get',
+      method: params.method ? params.method.toUpperCase() : 'GET',
```

Comment:
Is it matter?

Reply:
Weirdly enough, it does. On my browser at least (Mozilla), although I'm not sure if it's browser related or not. 

Basically, the CORS headers were not set properly with PATCH method when calling the fetch method with `"patch"` in lower cases, it would set the `Access-Control-Request-Method` to  `patch` instead of `PATCH` and the `OPTIONS` call would fail.

So this does the trick and, I think, should not affect anything as the actual methods are upper cases in the end.

Reply:
Cool. Good to know :)